lua/configs/which-key-register.lua

```diff
@@ -8,6 +8,7 @@ local mappings = {
       f = { name = "File" },
       p = { name = "Packer" },
       l = { name = "LSP" },
+      u = { name = "User" },
```

Comment:
I think naming this to "User Interface" or even "UI" would be more clear what the menu options do.